src/Pages/Liked/Liked.css

```diff
@@ -0,0 +1,10 @@
+.msg-container{
+    width: 30rem;
+    height: 5rem;
+    /* background-color: var(--vl-primary); */
```

Comment:
You can remove commented CSS if it's no longer needed.

Reply:
thanks for pointing it out, I have made the required changes.